addons/api/.stylelintignore

```diff
@@ -0,0 +1,8 @@
+# unconventional files
+/blueprints/*/files/
+
+# compiled output
+/dist/
+
+# addons
+/.node_modules.ember-try/
```

Comment:
is this part of the upgrade?

Reply:
Yeah it was automatically added as part of the upgrade, I can actually remove stylelint since we shouldn't ever need it in the API addon